### DISCUSSION TITLE
Button color 可以加个灰色背景色吗

### DISCUSSION THREAD

**Discussion by zhushenhang** [ant-design/ant-design-mobile#5186] [Q&A] [answered]
有个需求，在按钮为不可用的时候，背景色也要置为灰色。
目前我是这样实现的
`                  <Button
                            color={!isExpired && 'primary'}
                            disabled={isExpired}
                            style={{
                                backgroundColor: isExpired && '#888888'
                            }}
                        >
                            {isExpired ? '已过期' : '我要报名'}
                        </Button>`

**Comment by awmleer** [comment, marked as answer]
目前按钮默认的逻辑是禁用时透明度变化，而不是变成灰色，所以只能是你发的这样强制改样式了

**Reply by zhushenhang** [reply]
好的
